addons/point_of_sale/tools/posbox/configuration/wireless_ap.sh

```diff
@@ -4,6 +4,15 @@ FORCE_HOST_AP="${1}"
 WIRED_IP=$(python3 -c "import netifaces as ni; print(ni.ifaddresses('eth0').get(ni.AF_INET) and ni.ifaddresses('eth0')[ni.AF_INET][0]['addr'] or '')")
 WIFI_NETWORK_FILE="/home/pi/wifi_network.txt"
 
+# Do we have to use the NetworkManager ?
+current_iotbox_version=$(cat "/var/odoo/iotbox_version")
+required_version="23.11"
+if [[ "$current_iotbox_version" == "$required_version" ]] || [[ "$current_iotbox_version" > "$required_version" ]]; then
```

Comment:
You can just use `"$current_iotbox_version" < "$required_version"`
By this way not needed an else